client/app/src/main/java/com/microsoft/research/karya/ui/registration/CreationCodeFragment.kt

```diff
@@ -109,22 +130,23 @@ class CreationCodeActivity : BaseActivity(useAssistant = true) {
                         creationCodeStatusIv.setImageResource(0)
                         creationCodeStatusIv.setImageResource(R.drawable.ic_baseline_check_circle_outline_24)
                         WorkerInformation.creation_code = creationCode
-                        startActivity(Intent(applicationContext, PhoneNumberActivity::class.java))
+                        findNavController().navigate(R.id.action_creationCodeFragment_to_phoneNumberFragment)
                     }
                 } else {
                     uiScope.launch {
```

Comment:
You won't need this once you remove `ioScope`, so remove this too.

Reply:
Sure